src/index.css

```
@@ -104,6 +105,65 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-bottom: 8px;
+}
+
+nav a:hover,
+a:focus {
+  background-color: #797876;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: 66px;
+  padding-top: 30px;
+  padding-left: 16px;
+  padding-right: 16px;
```

This can be shortened to one prop if you like:

```suggestion
  padding: 30px 16px 66px;
```